lib/NodeDomain.js

```diff
@@ -32,6 +32,11 @@ define(function (require, exports, module) {
         EventEmitter = require("eventEmitter");
 
     var NodeConnection = require("./NodeConnection");
+
+    /**
+     * @define{number} Milliseconds to wait before giving up trying to validate existing domain.
```

Comment:
I'm going with the "when in rome" argument.  I think it should stay `@define`, similar to the analogs in NodeConnection, and then we should change them all if you feel strongly